locales/en/app.ftl

```diff
@@ -989,3 +989,6 @@ floating-banner-dismiss-button-label = No thanks
 banner-monitor-rebrand-text = <b>{ -brand-mozilla-monitor }</b>: New name, look and even more ways to <b>reclaim your privacy</b>.
 banner-monitor-rebrand-dismiss-button-label = OK
 banner-monitor-rebrand-dismiss-button-tooltip = Dismiss
+
+loading-sr-announcement = Loading
```

Comment:
What does `sr` stand for here? Maybe `accessibility` instead of `announcement` would avoid confusion.